django/db/backends/oracle/compiler.py

```diff
@@ -42,7 +42,36 @@ def as_sql(self, with_limits=True, with_col_aliases=False, subquery=False):
 
 
 class SQLInsertCompiler(compiler.SQLInsertCompiler, SQLCompiler):
-    pass
+    def as_sql(self):
```

Comment:
Ok.. I am closing this ticket then. I will do another pull request with solution suggested by issue reporter.
Thanks for advise!